index.html

```diff
@@ -248,12 +239,59 @@ <h2>Skills</h2>
             </div>
           </li>
         </ul>
+      </section>
+      <section class="get-in-touch">
+        <h3 class="intro-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+<form action="https://formspree.io/f/mpzblelz" method="post">
+
+<ul>
+
+<li>
+<input type="text" required value="First Name" maxlength="30" name="user_name">
+</li>
+<li>
+  <input required type="email" value="Email address" name="user_email" >
+</li>
```

Comment:
- [ ] Kindly use the `placeholder` attribute instead of the `value` attribute. So when the user starts typing the text disappears for a better user experience.